services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Remove and fix issues?

Reply:
@mmumby did you add this in the PR I merged?

Reply:
Apparently, I did. Will fix all linting in my follow up PR (Which will be up soon)